src/main/java/com/orgzly/org/OrgPatterns.java

```diff
@@ -42,4 +42,7 @@
     public static final Pattern HEAD_TAGS_P = Pattern.compile("^(.*)\\s+:(\\S+):\\s*$");
 
     public static final Pattern PROPERTY = Pattern.compile("^:([^:\\s]+):\\s+(.*)\\s*$");
+
+    public static final Pattern LOGBOOK_NOTE_P = Pattern.compile("^- Note taken on " + DT);
```

Comment:
@nevenz I don't like this regexp: in particular it doesn't match the full line. I couldn't seem to add anything after `DT`, that would cause the regexp to fail matching everything. Any ideas?